src/liquid/app/nav.liquid

```diff
@@ -80,8 +80,8 @@
       
         <div class="navbar-dropdown is-right">
           {% if showListControls %}
-            {% include "components/button_upload.html" className="navbar-item" %}
-            {% include "components/button_save_credentials.html" className="navbar-item" %}
+            {% render "components/button_upload.liquid", className: "navbar-item" %}
```

Comment:
Inconsistent use of single and double quotation. Perhaps we can agree on this [style guide ](https://docs.communityhealthtoolkit.org/community/contributing/code/style-guide/#use-single-quotes)

Reply:
Will update. The logic can follow this style guide, but any html attribute still needs double quotes.